src/ics/converter/types/calendar.py

```diff
@@ -62,4 +68,27 @@ def serialize(self, component: Component, output: Container, context: ContextDic
         context["VTIMEZONES_AFTER"] = len(output)
 
 
+class CalendarNameConverter(GenericConverter):
+    @property
+    def priority(self) -> int:
+        return 1100
+
+    @property
+    def filter_ics_names(self) -> List[str]:
+        return ["NAME", "X-WR-CALNAME"]
+
+    def populate(self, component: Component, item: ContainerItem, context: ContextDict) -> bool:
+        if item.name == "NAME":
+            return True
+        if item.name == "X-WR-CALNAME":
+            if component.name is not None:
+                return False
+
+            component.name = item.value
+            return False
```

Comment:
Actually I think we do want that because both google and apple do not handle NAME but only X-WR-CALNAME. I tested this locally and if you have NAME it does not care. It's unfortunate that they do not follow the RFC...